File: projektvorlage/go/features/vorlage/handlers_test.go
// Test der Handler. Laeuft unveraendert gruen.
package template

import "testing"

func TestBautDasModell(t *testing.T) {
	got, err := Handle(7)
	if err != nil {
		t.Fatalf("unerwarteter Fehler: %v", err)
	}
	if got.Value != 7 {
		t.Errorf("Handle(7).Value = %d, erwartet 7", got.Value)
	}
}

File: projektvorlage/typescript/features/_template/handlers_test.ts
// Test der Handler. Laeuft unveraendert gruen.

import { test } from "node:test";
import assert from "node:assert/strict";

import { handle } from "./handlers.ts";

test("baut das modell", () => {
  assert.equal(handle(7).value, 7);
});
